api/client/lib.py

```diff
@@ -513,6 +516,8 @@ def list_of_series_to_single_series(series_list, add_belongs_to=False):
                 'frequency_id': series['series'].get('frequencyId', None)
                 # 'source_id': series['series'].get('sourceId', None), TODO: add source to output
             }
+            if series_metadata:
+                formatted_point['metadata'] = series_metadata
```

Comment:
This would duplicate the series-level metadata onto every single point and looks like point-level metadata but isn't. Not sure if or how we should differentiate series-level vs. point-level. Not important to add this metadata at the moment, right? Maybe just exclude this bit for now until we have a clear spec on what that should look like.

Reply:
good point, removing from point level formatting